requirements/notebook-ci.txt

```diff
@@ -1,7 +1,7 @@
 neo4j >= 5.0, < 6.0
 pyarrow >= 10.0, < 11.0
 pytest
-nbconvert == 7.2.8
+nbconvert == 7.2.9
```

Comment:
i would have hoped dependabot catches this. Likely the bot is not working at the moment

Reply:
Should it always notice us on any patch release? Or just important releases with security fixes?

Reply:
sounds good to create a card

Reply:
I might have found the problem. Maybe the directory spec was wrong

Reply:
ah ok :+1: won't add a card then